### PR TITLE
Feature/mission bag buoys

### PR DESCRIPTION
**Pull request by kaowilly**
**Warning :** Before creating a pull request, make sure that it does not already exists in the [pull requests](../). Thank you.

## Description
Added 2 behaviors: mission_bags_bins and mission_bags_buoys

Those missions need to be tweaked before the next pool test.

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      ],
+      "name": "ROS",
+      "intelliSenseMode": "gcc-x64",
+      "compilerPath": "/usr/bin/gcc",
+      "cStandard": "gnu11",
+      "cppStandard": "c++14"
+    }
+  ],
+  "version": 4
+}
```
Comment (Kam0ss) [inline .vscode/c_cpp_properties.json]: This file should not be here, there is no C++ in this node.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "python.autoComplete.extraPaths": [
+        "/home/willy/ros_sonia_ws/devel/lib/python2.7/dist-packages",
+        "/opt/ros/melodic/lib/python2.7/dist-packages"
+    ],
+    "python.analysis.extraPaths": [
+        "/home/willy/ros_sonia_ws/devel/lib/python2.7/dist-packages",
+        "/opt/ros/melodic/lib/python2.7/dist-packages"
+    ]
+}
```
Comment (Kam0ss) [inline .vscode/settings.json]: This file should not be here, it will make problems on another computer.

**Review by Kam0ss — CHANGES_REQUESTED** [review]
Just delete .vscode files

File: .vscode/c_cpp_properties.json (hunk 1 above)
Comment (kaowilly) [reply .vscode/c_cpp_properties.json]: File deleted as requested.

File: .vscode/settings.json (hunk 2 above)
Comment (kaowilly) [reply .vscode/settings.json]: File deleted as requested.

**Review by Kam0ss — APPROVED** [review]
LGTM

**Pull request merged**